_posts/0000-01-02-sara88888.md

```diff
@@ -0,0 +1 @@
+console.log('Olá')
```

Comment:
layout: slide
title: "Welcome to our second slide!"
---
Your text
Use the left arrow to go back!